src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/AssertExtensions.cs

```diff
@@ -438,6 +440,15 @@ public static void AtLeastOneEquals<T>(T expected1, T expected2, T value)
             {
                 throw new ThrowsException(typeof(E), exception);
             }
+
+            return exception;
+        }
+
+        public static void Throws<E, T>(string expectedParamName, ReadOnlySpan<T> span, AssertThrowsActionReadOnly<T> action)
```

Comment:
The other `Throws(expectedParamName` methods return the exception; might as well go ahead and do it here for consistency (of type `E`).

And it probably makes sense to add the writable-span version for symmetry.